### ISSUE TITLE
Quotes in parameters in step definition

### ISSUE THREAD

**Issue by CameronSabuda** [Arthy000/gherkin-testcafe#95]
On a project that I work on, we have been wrapping the parameters in our step definitions in double quotes so that they are more obvious when written in our feature files like this:
```cucumber
  Scenario: Scenario name here
        Given I am logged in as an Admin
        When I go to the website
        And I click the "insert link name here" link
        And I fill out the form using numbers
        And I click the "Save" button
        Then I will see an item with the Display Name of "1234"
```

However, when the parameter has double quotes in it, it fails to find the step definition despite it still matching the regex in the step definition.

Step definition:
```js
Then(/^I will see the "(.*)" graphql error message$/, async (t, [entry]) => {
```

Feature file:
```
Then I will see the "Item with id "insert_id_here" already exists" graphql error message
```

Output:
```
1) Error: Step implementation missing for: I will see the "Item with id "insert_id_here" already exists" graphql error message
```

**Comment by Arthy000** [comment]
Hi @CameronSabuda,

I tried several things, including copypasting your example, however I can't reproduce the behavior you described.

Feature file:
```gherkin
Scenario: testing "quoted "param" here"
    Given "quoted "param" here" is between quotes
    Then I will see the "Item with id "insert_id_here" already exists" graphql error message
```
Step file:
```ts
Given(/"(.+)" is between quotes/, async (t: TestController, [param]) => {
  console.log("PARAM", param);
});

Then(/^I will see the "(.*)" graphql error message$/, async (t, [entry]) => {
  await t.navigateTo("http://google.be").wait(3000);
  console.log("ENTRY:", entry);
});
```
Output:
```
PARAM quoted "param" here
ENTRY: Item with id "insert_id_here" already exists
 ✓ Scenario: testing "quoted "param" here"
```
Everything seems to match as expected.

Could you tell me which version of cucumber and gherkin-testcafe you are using?

_On a personal note, I find that using nested double quotes is not really helping readability, regardless of the current issue. Have you considered using single quotes instead?_

**Comment by CameronSabuda** [comment]
Interesting, I've just tried it all again and now it all works perfectly 😅

We recently did a bunch of package updates so I assume one of the packages somewhere was updated and may have fixed this bug? Not sure, but I don't believe it's an issue anymore and I'm sorry to have wasted your time! But thank you for taking the time to investigate.

In regards to your final question, the reason we had chosen to use double quotes for that specific step was purely because we have a rather large amount of test fixtures in our testing suite with many tests using variables marked by being in double quotes. If we used single quotes for this specific test definition just because the variable in question contained double quotes then it would have been confusing and changing everything to single quotes would've probably also caused confusion to the other developers
